api-admin/src/main/java/bio/terra/pearl/api/admin/service/portal/PortalExtService.java

```diff
@@ -60,7 +60,7 @@ public PortalEnvironmentConfig updateConfig(
       throw new PermissionDeniedException(
           "You do not have permissions to update portal configurations");
     }
-    Portal portal = authUtilService.authUserToPortal(user, portalShortcode);
+    authUtilService.authUserToPortal(user, portalShortcode);
```

Comment:
Interesting -- to me a naked `get()` is what communicates that "if it fails, a deep invariant has been violated". And to me the code is more readable than adding a bunch of "orElseThrow(SomeInvariantViolatedException)". That said, I think your take is more correct as the codebase scales in usage and team size since it always identifies which invariant.  The question for us is that given our team size and turnover, and since those invariant code exceptions will rarely get exercised/tested, is adding them worth the maintenance costs (since those paths would have to be tested as otherwise they'd be a prime source of copypasta error.)

Reply:
A naked `get()` to me is ambiguous: careless coding or an invariant?
I don't think I have a strong opinion about how to indicate an invariant (old school asserts did that), but I feel pretty strongly that a well understood way of expressing that is important.

Reply:
Reading https://stackoverflow.com/questions/49158806/optional-get-versus-overloaded-optional-orelsethrow has confirmed my suspicion that there are a large number of folks like me who prefer `get()` 
```
in Java 10 we proposed to deprecate Optional.get(), but the public response to that was ... let's say cold.
```
but that your view is seen as the correct one.  So I'll give it a shot.  Created some new exception classes and switched to orElseThrow